app/enterprise/2.4.x/deployment/installation/docker.md

```diff
@@ -38,79 +38,67 @@ To complete this installation you will need a Docker-enabled system with proper
   -e "KONG_PROXY_ERROR_LOG=/dev/stderr" \
   -e "KONG_ADMIN_ERROR_LOG=/dev/stderr" \
   -e "KONG_ADMIN_LISTEN=0.0.0.0:8001" \
-  -e "KONG_ADMIN_GUI_URL=http://<div contenteditable="true">{DNS_or_IP}</div>:8002" \
-  -p 8000:8000 \
-  -p 8443:8443 \
-  -p 8001:8001 \
-  -p 8444:8444 \
-  -p 8002:8002 \
-  -p 8445:8445 \
-  -p 8003:8003 \
-  -p 8004:8004 \
-  kong-ee </code></pre>
-
-<div class="alert alert-ee">
-<b>Note:</b> For <code>KONG_ADMIN_GUI_URL</code>, replace <code>&lt;DNSorIP&gt;</code>
-with with the DNS name or IP of the Docker host. <code>KONG_ADMIN_GUI_URL</code>
-<i>should</i> have a protocol, for example, <code>http://</code>.
-</div>
+  -e "KONG_ADMIN_GUI_URL=http://<div contenteditable="true">{DNS_OR_IP}</div>:8002" \
+    -p 8000:8000 \
+    -p 8443:8443 \
+    -p 8001:8001 \
+    -p 8444:8444 \
+    -p 8002:8002 \
+    -p 8445:8445 \
+    -p 8003:8003 \
+    -p 8004:8004 \
+    kong-ee</code></pre>
+
+{:.note}
+> The DNS or IP address for `KONG_PORTAL_GUI_HOST` should be preceded by a protocol, for example, `http://`.
 
 ## Step 6. Verify your installation
 
 1. Access the `/services` endpoint using the Admin API:
 
-    <pre><code>curl -i -X GET --url http://<div contenteditable="true">{HOSTNAME}</div>:8001/services</code></pre>
+    <pre><code>curl -i -X GET --url http://<div contenteditable="true">{DNS_OR_IP}</div>:8001/services</code></pre>
```

Comment:
Can you change all the `{DNS_OR_IP}` entries to `{HOSTNAME}` (or just `{HOST}`)?

Reply:
Changed.